src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html

```diff
@@ -15,11 +15,11 @@
           <th:block th:ref="formContent">
 
             <div class="form-card__content">
-              <div th:each="individual, iter: ${inputData.get('pregnanciesFullNames[]')}" class="spacing-below-15">
+              <div th:each="individual, iter: ${inputData.get('pregnanciesFullNames[]')}" class="spacing-below-15" th:with="messageVariant=${individual.uuid == 'you' ? 'self' : 'householdMember'}">
```

Comment:
We'll always have a "you" so I think you can pull this out of the iterator, then only iterate on householdMembers

Reply:
Oh ignore that - this is dependent on `pregnancies` 🤦